src/metric_config.cpp

```diff
@@ -112,7 +112,7 @@ void MetricConfig::setDefaultMetricsTo(bool enabled) {
     }
 }
 
-Status MetricConfig::loadFromCLIString(bool isEnabled, const std::string& metricsList, uint64_t restPort) {
+Status MetricConfig::loadFromCLIString(bool isEnabled, const std::string& metricsList, bool failIfMetricsAreEnabled) {
```

Comment:
Why do we need `failIfMetricsAreEnabled` here? We can pass `false` in line 143, right?

Reply:
If so, use default parameter

Reply:
Done.